src/main/java/com/rapid7/net/Utils.java

```diff
@@ -0,0 +1,15 @@
+package com.rapid7.net;
+
+final class Utils {
+  // private constructor
+  private Utils (){}
+  
+  /**
+   * Check if the contained string is <code>null</code> or empty. Trimming is NOT applied.
+   * @param s The String to be checked.
+   * @return <code>true</code> if the passed String is <code>null</code> or empty.
+   */
+  public static boolean isNullOrEmpty (String s){
```

Comment:
looks great, just this and if you could bump patch in `pom.xml`
```suggestion
  public static boolean isNullOrEmpty (final String s){
```

Reply:
I bumped the POM version to 1.1.41 but that is in conflict 🤷‍♂

Reply:
@phax we released `3.0.0` yesterday, so it will need to be `3.0.1` now I'm afraid

Reply:
You'll need to rebase on top of the latest also

Reply:
Okay - I will do it after my vacation. Happy holidays